react/features/base/dialog/components/native/styles.js

```diff
@@ -135,45 +138,53 @@ export const inputDialog = {
  * {@link https://material.io/guidelines/components/bottom-sheets.html}.
  */
 ColorSchemeRegistry.register('BottomSheet', {
-    /**
-     * Style for the {@code Icon} element in a generic item of the menu.
-     */
-    iconStyle: {
-        color: schemeColor('icon'),
-        fontSize: 24
+    buttons: {
```

Comment:
just doublw-checking: this change doesn't require any changes to any branding, right?

Reply:
No, I just wanted to separate the AbstractButton styles. The branding gets applied recursively